upstream-community-operators/kubevirt/kubevirtoperator.0.17.0-alpha.0.csv.yaml

```diff
@@ -0,0 +1,566 @@
+              - subresources.kubevirt.io
+              resources:
+              - virtualmachineinstances/console
+              - virtualmachineinstances/vnc
+              verbs:
+              - get
+            - apiGroups:
+              - subresources.kubevirt.io
+              resources:
+              - virtualmachines/restart
+              verbs:
+              - put
+              - update
+            - apiGroups:
+              - kubevirt.io
+              resources:
+              - virtualmachines
+              - virtualmachineinstances
+              - virtualmachineinstancepresets
+              - virtualmachineinstancereplicasets
+              verbs:
+              - get
+              - delete
+              - create
+              - update
+              - patch
+              - list
+              - watch
+              - deletecollection
+            - apiGroups:
+              - subresources.kubevirt.io
+              resources:
+              - virtualmachineinstances/console
+              - virtualmachineinstances/vnc
+              verbs:
+              - get
+            - apiGroups:
+              - subresources.kubevirt.io
+              resources:
+              - virtualmachines/restart
+              verbs:
+              - put
+              - update
+            - apiGroups:
+              - kubevirt.io
+              resources:
+              - virtualmachines
+              - virtualmachineinstances
+              - virtualmachineinstancepresets
+              - virtualmachineinstancereplicasets
+              verbs:
+              - get
+              - delete
+              - create
+              - update
+              - patch
+              - list
+              - watch
+            - apiGroups:
+              - kubevirt.io
+              resources:
+              - virtualmachines
+              - virtualmachineinstances
+              - virtualmachineinstancepresets
+              - virtualmachineinstancereplicasets
+              verbs:
+              - get
+              - list
+              - watch
+            - apiGroups:
+              - authentication.k8s.io
+              resources:
+              - tokenreviews
+              verbs:
+              - create
+            - apiGroups:
+              - authorization.k8s.io
+              resources:
+              - subjectaccessreviews
+              verbs:
+              - create
+
+      deployments:
+        - name: virt-operator
+          spec:
+            replicas: 2
+            selector:
+              matchLabels:
+                kubevirt.io: virt-operator
+            strategy:
+              type: RollingUpdate
+            template:
+              metadata:
+                annotations:
+                  scheduler.alpha.kubernetes.io/critical-pod: ""
+                  scheduler.alpha.kubernetes.io/tolerations: '[{"key":"CriticalAddonsOnly","operator":"Exists"}]'
+                creationTimestamp: null
```

Comment:
nit: Please remove the `creationTimestamp` field from the operator deployment spec

Reply:
Fixed